app/routes/events/components/EventDetail/index.js

```diff
@@ -295,26 +288,58 @@ export default class EventDetail extends Component<Props> {
               </Flex>
             </ContentMain>
             <ContentSidebar>
-              <InfoList items={infoItems} />
+              {event.company && (
+                <div className={styles.iconWithText}>
+                  <Icon name="briefcase-outline" className={styles.infoIcon} />
+                  <strong>
+                    <Link to={`/companies/${event.company.id}`}>
+                      {event.company.name}
+                    </Link>
+                  </strong>
+                </div>
+              )}
+              <div className={styles.iconWithText}>
+                <Icon name="time-outline" className={styles.infoIcon} />
+                <strong>
+                  <FromToTime from={event.startTime} to={event.endTime} />
+                </strong>
+              </div>
               {event.isPriced && (
-                <div className={styles.paymentInfo}>
-                  <strong>Dette er et betalt arrangement</strong>
-                  <InfoList items={paidItems} />
+                <div className={styles.iconWithText}>
+                  <Icon name="cash-outline" className={styles.infoIcon} />
+                  <strong>{event.priceMember / 100},-</strong>
                 </div>
               )}
+              <div className={styles.iconWithText}>
+                <Icon name="pin-outline" className={styles.infoIcon} />
+                <strong>{event.location}</strong>
+              </div>
               {event.mazemapPoi && (
-                <MazemapEmbed mazemapPoi={event.mazemapPoi} />
+                <>
+                  <div
+                    className={styles.simulateLink}
+                    onClick={() =>
+                      this.setState({ mapIsOpen: !this.state.mapIsOpen })
+                    }
+                  >
+                    {' '}
+                    Vis kart
+                  </div>
+                  {this.state.mapIsOpen && (
+                    <MazemapEmbed mazemapPoi={event.mazemapPoi} />
+                  )}
+                </>
               )}
               {['OPEN', 'TBA'].includes(event.eventStatusType) ? (
                 <JoinEventForm event={event} />
               ) : (
-                <Flex column>
+                <Flex column className={styles.registeredBox}>
```

Comment:
Thoughts about putting a `<Line />` in here to make the separation of data groups consistent?

Reply:
As we previously discussed, I think adding another line here will just clutter everything more and will look a bit out of place.
I think the headline here acts as more than enough separation as well.

If anything, we could _remove_ some of the lines above and below the info boxes to duclutter and clean up a bit. I'm open for suggestions.